examples/plotting/dynamic_color_mapping.py

```diff
@@ -1,3 +1,10 @@
+''' This example shows how to use colours to map range of values within a given
+array as low, high or median.
```

Comment:
I don't follow the reference to medians, the linear color mapper does not use any median computations. Maybe just:
```suggestion
array.
```

Reply:
Hi @bryevdv, I think it does although I may have not explained correctly.

For instance, we have an array for numbers between 1 - 100 and colours red, blue and green. Numbers within the range 1-40 will be given red, numbers within the range 41 - 60 can be given blue and numbers within the range 61-100 will be given green. 

So using the color mapper, we have been able to show, maybe with a scatterplot the distribution of the numbers when plotted.

Waiting for your feedback on this. Thanks for the rest, would make the correct edits and push it.

Reply:
Hi @Richiio when I see "median", I only think of the mathematical definition: 

>noun
Arithmetic, Statistics. the middle number in a given sequence of numbers, taken as the average of the two middle numbers when the sequence has an even number of numbers:
4 is the median of 1, 3, 4, 8, 9.

But this usage in this example has nothing to do with that, so  do think there is room for confusion. I would still request applying the suggestion. (I think "use colours to map range of value" is sufficient to express the main point)